homeassistant/components/netatmo/climate.py

```diff
@@ -328,6 +328,35 @@ def device_state_attributes(self):
 
         return attr
 
+    def turn_off(self):
+        """Turn the entity off."""
+        if self._module_type == NA_VALVE:
+            self._data.homestatus.setroomThermpoint(
+                self._data.home_id,
+                self._room_id,
+                STATE_NETATMO_MANUAL,
+                DEFAULT_MIN_TEMP,
+            )
+        elif self.hvac_mode != STATE_OFF:
+            self._data.homestatus.setroomThermpoint(
+                self._data.home_id, self._room_id, HVAC_MODE_OFF
+            )
+        self.update_without_throttle = True
+        self.schedule_update_ha_state()
+
+    def turn_on(self):
+        """Turn the entity on."""
+        self._data.homestatus.setroomThermpoint(
+            self._data.home_id, self._room_id, STATE_NETATMO_HOME
+        )
+        self.update_without_throttle = True
+        self.schedule_update_ha_state()
+
+    @property
+    def available(self) -> bool:
+        """If the device hasn't been able to connect, mark as unavailable."""
+        return self._connected or self._connected is None
```

Comment:
If `self._connected` is `None` the entity should be unavailable, right?

Reply:
That is correct, thank you.